chatmessage/test_chatmessage.py

```diff
@@ -0,0 +1,49 @@
+from .models import Chatmessage
+from account.models.professional import Professional
+from account.models.client import Client
+import pytest
+
+
+@pytest.mark.django_db
+def test_chat_response_code(client):
```

Comment:
Please add template check here, that relevant template returned

Reply:
Done